apps/interactor/src/components/interactor/Interactor.scss

```
@@ -42,15 +42,18 @@
     }
   }
 
-  &__background-image {
-    height: 100%;
-    width: 100%;
-    z-index: 0;
-    border-radius: 0.75rem; // Equivalent to 'rounded-xl' in Tailwind
-    object-fit: cover;
-    transition: all 0.2s ease-in;
-    pointer-events: none;
-    user-select: none;
+  &__background {
+    &-image,
+    &-video {
```

don't nest that many items, it's hard to read
```suggestion
  &__background-image,
  &__background-video {
```